exercises/robot-simulator/robot_simulator_test.py

```diff
@@ -3,18 +3,23 @@
 from robot_simulator import Robot, NORTH, EAST, SOUTH, WEST
 
 
-# Tests adapted from `problem-specifications//canonical-data.json` @ v3.0.0
+# Tests adapted from `problem-specifications//canonical-data.json` @ v3.1.0
 
 class RobotSimulatorTest(unittest.TestCase):
+    # Case1: A robot is created with a position and a direction
+    #  - Robots are created with a position and direction
```

Comment:
I'm not sure when the current test function names were introduced, but they should really reflect the test descriptions directly.

Example:
`def test_robot_created_with_position_and_direction(self):`

If the name gets too long, paraphrase. If there is no paraphrase that works, get as close as possible, then add a comment for clarification.